docs/book/src/topics/multitenancy.md

```diff
@@ -112,17 +114,19 @@ stringData:
 The assumed role could be used by the AWSClusters that is in the `allowedNamespaces`.
 
 Example:
-Below, an `AWSClusterRoleIdentity` instance, which will be used by AWSCluster "test", is created.
+Below, an `AWSClusterRoleIdentity` instance, which will be used by `AWSCluster` "test", is created.
 This role will be assumed by the source identity at runtime. Source identity can be of any identity type.
 Role is assumed in the beginning once and after, whenever the assumed role's credentials are expired.
 
+This snippet illustrates the connection between `AWSCluster`and the `AWSClusterRoleIdentity`, however this is not a working example.
+Please vew a full example below.
+
 ```yaml
 ---
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: AWSCluster
 metadata:
   name: "test"
-  namespace: "test"
```

Comment:
What's the reason to remove the namespace here?

Reply:
Bad copy / paste, putting back. Thanks for catching this.